site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -16,6 +45,27 @@
     "assets/js/trumbowyg.min.js"
   %}
     <script src="{{ asset_url }}"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script>
+      $(document).ready(function(){
+      $("#delete-project-text").keyup(function(){
+          var projectName = $("#project_displayName").val();
+          var matched = false;
+          if($(this).val() != projectName){
```

Comment:
this can be written as `matched = $(this).val() != projectName;` or even better, combine it with the if-clause below to give you

```js
if ($(this).val() != projectName) {
  $('#delete-project-button').attr('disabled', 'disabled');
} else {
  $('#delete-project-button').removeAttr('disabled');
}

Reply:
Made the requested changes.